diff --git a/node_modules/react-native-web/dist/exports/Text/index.js b/node_modules/react-native-web/dist/exports/Text/index.js
index 8c5f79b..124f7b6 100644
--- a/node_modules/react-native-web/dist/exports/Text/index.js
+++ b/node_modules/react-native-web/dist/exports/Text/index.js
@@ -26,7 +26,8 @@ import { warnOnce } from '../../modules/warnOnce';
 var forwardPropsList = Object.assign({}, forwardedProps.defaultProps, forwardedProps.accessibilityProps, forwardedProps.clickProps, forwardedProps.focusProps, forwardedProps.keyboardProps, forwardedProps.mouseProps, forwardedProps.touchProps, forwardedProps.styleProps, {
   href: true,
   lang: true,
-  pointerEvents: true
+  pointerEvents: true,
+  draggable: true,
 });
 var pickProps = props => pick(props, forwardPropsList);
 var Text = /*#__PURE__*/React.forwardRef((props, forwardedRef) => {
diff --git a/node_modules/react-native-web/dist/exports/View/index.js b/node_modules/react-native-web/dist/exports/View/index.js
index c812d77..a444ace 100644
--- a/node_modules/react-native-web/dist/exports/View/index.js
+++ b/node_modules/react-native-web/dist/exports/View/index.js
@@ -26,7 +26,8 @@ var forwardPropsList = Object.assign({}, forwardedProps.defaultProps, forwardedP
   lang: true,
   onScroll: true,
   onWheel: true,
-  pointerEvents: true
+  pointerEvents: true,
+  draggable: true
 });
 var pickProps = props => pick(props, forwardPropsList);
 var View = /*#__PURE__*/React.forwardRef((props, forwardedRef) => {
diff --git a/node_modules/react-native-web/src/exports/Text/index.js b/node_modules/react-native-web/src/exports/Text/index.js
index 071ae10..e22e9ab 100644
--- a/node_modules/react-native-web/src/exports/Text/index.js
+++ b/node_modules/react-native-web/src/exports/Text/index.js
@@ -37,7 +37,8 @@ const forwardPropsList = Object.assign(
   {
     href: true,
     lang: true,
-    pointerEvents: true
+    pointerEvents: true,
+    draggable: true
   }
 );
 
